usr/src/test/zfs-tests/include/default.cfg

```diff
@@ -88,6 +88,11 @@ export TESTSUBFS=subfs
 export TESTSUBFS1=subfs1
 export TESTSUBFS2=subfs2
 
+# some test sub file system names
+export TESTSUBFS=subfs.$$
+export TESTSUBFS1=subfs1.$$
+export TESTSUBFS2=subfs2.$$
+
```

Comment:
The trailing ".$$" on these files isn't necessary, and make it more difficult to do cleanup of ad hoc testing. They've been removed from the Delphix version.